README.md

```diff
@@ -1,57 +1,17 @@
-# Northcoders News API
+BE-NC-NEWS
 
-## Background
+Hi all, welcome to my repo!
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+I be building an API for the purpose of accessing application data programmatically.
+My intention here is to mimic the building of a real world backend service (such as reddit, quora, YOU KNOW THE SCORE!) which should provide this information to the front end architecture.
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+Connect Locally
+You can either fork this repo and then make your own clone of this repository and run the command npm i to get started.
```

Comment:
This might just need tidying up as I'm not sure what the other thing is that the either refers to!